src/App.tsx

```diff
@@ -1,14 +1,72 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { PersonList } from './components/PersonList';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const { name, born, died } = selectedPerson || {};
+  const [onFocus, setOnFocus] = useState(false);
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [hasListVisible, setHasListVisible] = useState(false);
+
+  let preparedPeopleList: Person[] = [...peopleFromServer];
```

Comment:
Can you answer, please, why u need it and why use let in this case?

Reply:
I use let because later I could change this array with filter method and put result to the same array.